package.json

```diff
@@ -20,6 +20,8 @@
     "@types/react": "^18.2.21",
     "@types/react-dom": "^18.2.7",
     "astro": "^3.0.12",
+    "cnbuilder": "^3.1.0",
```

Comment:
The [cnbuilder package](https://www.npmjs.com/package/cnbuilder) isn't necessary but since we use it within our projects with decanter-react, I figured I'd include it for the example.